### DISCUSSION TITLE
Need Help To Uncompile python 3.11 file

### DISCUSSION THREAD

**Discussion by Kingkhan14** [rocky/python-uncompyle6#475] [General]
i want to uncompile a script that compiled by pyinstaller and python 3.11 version.
just became a sponsor of you is that enough to get my script decompiled.
please respond ASAP

thanks

**Comment by rocky** [comment]
Time to time I get requests like the above.  What tends to happen in with requests like this is that promises are as broken as easily as they are made. 

Therefore folks should know that in the future, payment will be required in advance of consultation or service.
